src/coreclr/jit/gentree.cpp

```diff
@@ -25286,6 +25286,17 @@ bool GenTreeHWIntrinsic::OperIsCreateScalarUnsafe() const
     }
 }
 
+//------------------------------------------------------------------------
+// OperIsBitwiseHWIntrinsic: Is this HWIntrinsic a bitwise logic intrinsic node.
+//
+// Return Value:
+//    Whether "this" is a bitwise logic intrinsic node.
+//
+bool GenTreeHWIntrinsic::OperIsBitwiseHWIntrinsic()
+{
+    return HWOperGet() == GT_AND || HWOperGet() == GT_OR || HWOperGet() == GT_XOR || HWOperGet() == GT_AND_NOT;
```

Comment:
nit: It'd be nice to explicitly cache `HWOperGet()`. The compiler "should" be doing it for us, but its not a trivial call and is better to be safe.